python_modules/dagster/dagster/_experimental/staging_io_manager.py

```diff
@@ -0,0 +1,96 @@
+from typing import Any, Optional, Sequence, Union
+
+from dagster import InputContext, OutputContext
+from dagster._core.definitions.events import AssetMaterialization, AssetObservation
+from dagster._core.definitions.metadata import (
+    MetadataEntry,
+    PartitionMetadataEntry,
+    TextMetadataValue,
+)
+from dagster._core.events import AssetObservationData, DagsterEventType, StepMaterializationData
+from dagster._core.events.log import EventLogEntry
+from dagster._core.storage.io_manager import IOManager
+
+
+def get_asset_materialization_from_event_log_entry(
```

Comment:
IMO we should make this a property on `EventLogEntry`. We've already done so with `EventLogRecord`.

Reply:
cool can do in follow up

Reply:
decided to do this in a PR before this one and restack